cmake/CTestConfig.cmake.in

```diff
@@ -0,0 +1,91 @@
+#----------------------------------------------------------------------------------------#
+#
+#   CTestConfig.cmake template for Kokkos
+#
+#----------------------------------------------------------------------------------------#
+
+#
+#   dash-board related
+#
+set(CTEST_PROJECT_NAME "Kokkos")
+set(CTEST_NIGHTLY_START_TIME "01:00:00 UTC")
+set(CTEST_DROP_METHOD "https")
+set(CTEST_DROP_SITE "cdash.nersc.gov")
+set(CTEST_DROP_LOCATION "/submit.php?project=${CTEST_PROJECT_NAME}")
+set(CTEST_CDASH_VERSION "1.6")
+set(CTEST_CDASH_QUERY_VERSION TRUE)
+set(CTEST_SUBMIT_RETRY_COUNT "1")
+set(CTEST_SUBMIT_RETRY_DELAY "30")
+
+#
+#   configure/build related
+#
+set(CTEST_BUILD_NAME "@BUILD_NAME@")
+set(CTEST_MODEL "@MODEL@")
+set(CTEST_SITE "@SITE@")
+set(CTEST_CONFIGURATION_TYPE "@BUILD_TYPE@")
+set(CTEST_SOURCE_DIRECTORY "@SOURCE_REALDIR@")
+set(CTEST_BINARY_DIRECTORY "@BINARY_REALDIR@")
+
+#
+#   configure/build related
+#
+set(CTEST_UPDATE_TYPE "git")
+set(CTEST_UPDATE_VERSION_ONLY ON)
+# set(CTEST_GENERATOR "")
+# set(CTEST_GENERATOR_PLATFORM "")
+
+#
+#   testing related
+#
+set(CTEST_TIMEOUT "7200")
+set(CTEST_TEST_TIMEOUT "7200")
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_ERRORS "100")
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS "100")
+set(CTEST_CUSTOM_MAXIMUM_PASSED_TEST_OUTPUT_SIZE "104857600")
```

Comment:
This correspond to 102400KiB.
Out of curiosity, how large is our typical submission?

Reply:
Not large. This was some boilerplate I copied over and the extra zeros were left over from a submission issue I had w.r.t. it being too large. Turns out it was an issue with the ingress and I just never corrected it

Reply:
They only get large when you start attaching files like this:

https://cdash.nersc.gov/testDetails.php?test=205766&build=3208

Reply:
Reduced to 1 MB